### ISSUE TITLE
Replace quadprog with cvxpy?

### ISSUE THREAD

**Issue by fernandezc** [spectrochempy/spectrochempy#682]
### Is your feature request related to a problem?

Perhaps we could replace Quadprog, which doesn't seem to be actively maintained, with cvxpy https://www.cvxpy.org/examples/basic/quadratic_program.html.

@atravert What's your opinion on this? Would you like to give it a try?

**Comment by atravert** [comment]
Yes I will look at this. We only need quadratic programming for now, so maybe use directly OSQP and make it optional (only used for IRIS so far...) ?

**Comment by fernandezc** [comment]
Yes, that's a good idea. There is a python interface for OSQP available on conda-forge. 

I think it would be a good idea to implement it soon, to allow the use of python 3.11.
